daprdocs/content/en/reference/api/conversation_api.md

```diff
@@ -0,0 +1,63 @@
+---
+type: docs
+title: "Conversation API reference"
+linkTitle: "Conversation API"
+description: "Detailed documentation on the conversation API"
+weight: 1400
+---
+
+Dapr provides users with an abstraction layer to reduce the complexity of interacting with LLMS using either a new runtime method or HTTP endpoint.
```

Comment:
`Dapr provides users with an abstraction layer to reduce the complexity of interacting with LLMS using either an HTTP endpoint or SDK methods`.